lib/ui/painting/codec.cc

```diff
@@ -79,10 +84,79 @@ static sk_sp<SkImage> DecodeImage(fml::WeakPtr<GrContext> context,
   }
 }
 
+static bool needsResize(const double currentWidth,
+                        const double currentHeight,
+                        const double targetWidth,
+                        const double targetHeight) {
+  if (fuzzyEquals(currentWidth, kDoNotResizeImage) ||
```

Comment:
Should it be comparing targetWidth/Height?

Reply:
Good catch!